internal/domain/domain_test.go

```diff
@@ -0,0 +1,48 @@
+package domain
+
+import (
+	"net/url"
+	"testing"
+)
+
+func TestDomainFromRequest(t *testing.T) {
+	// create the test cases
```

Comment:
question: do we need those comments? Isn't this standard approach in Go to create table tests? As long as as we don't create some fancy custom mechanisms there comments may bring no additional details

Reply:
I am just used to do it like that. If we take the decision to omit these, we should go through all services as it is like that in many places.